c7n/filters/core.py

```diff
@@ -432,8 +432,7 @@ def process_value_type(self, sentinel, value):
                 # EMR not having more functionality.
                 try:
                     value = parse(value, default=datetime.now(tz=tzutc()))
-
-                except (AttributeError, TypeError):
+                except:
```

Comment:
dateutil.parse will raise exceptions like ValueError and other things commonly. It's probably better if we catch all exceptions instead of white listing two kinds of errors. I noticed one of my users put garbage in an 'ExpiresAt' tag that gets sent to this function.